source/_static/scss/includes/_aside.scss

```diff
@@ -20,6 +20,16 @@
     }
 }
 
+.index-link {
```

Comment:
Adding back the index link as per discussion, so users can easily find the index + know which docs property they are on.